arch/arm/soc/st_stm32/stm32f7/soc.c

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2018 Yurii Hamann
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief System/hardware module for STM32F7 processor
+ */
+
+#include <kernel.h>
+#include <device.h>
+#include <init.h>
+#include <soc.h>
+#include <arch/cpu.h>
+#include <cortex_m/exc.h>
+
+/**
+ * @brief This function configures the source of stm32cube time base.
+ *        Cube HAL expects a 1ms tick which matches with k_uptime_get_32.
+ *        Tick interrupt priority is not used
+ * @return HAL status
+ */
+uint32_t HAL_GetTick(void)
```

Comment:
Could you remove this function? It is now provided by arch/arm/soc/st_stm32/common/stm32cube_hal.c